docs/bugs.json

```diff
@@ -73,6 +74,7 @@
         "name": "SignedArrayStorageCopy",
         "summary": "Assigning an array of signed integers to a storage array of different type can lead to data corruption in that array.",
         "description": "In two's complement, negative integers have their higher order bits set. In order to fit into a shared storage slot, these have to be set to zero. When a conversion is done at the same time, the bits to set to zero were incorrectly determined from the source and not the target type. This means that such copy operations can lead to incorrect values being stored.",
+        "link": "https://blog.ethereum.org/2019/06/25/solidity-storage-array-bugs/",
```

Comment:
Should we change all these URLs to `solidity.ethereum.org` ?

Reply:
If they work, why not?

Reply:
cc @franzihei @chriseth

Reply:
I wonder if the URLs will stay stable, but we can always change it again in that list.

Concerning stability of the URLs: Maybe the blog posts should be under `solidity.ethereum.org/blog/date/post`